log_data.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+file_path = 'D:\Python\Repo\Training_Repo\log_data.txt'
+
+def top_downloader(path):
+    with open(file_path) as f:
+        calc_dict = dict()
+        for line in f:
+            ip_date = line.split()[1][:12] + '] ' + line.split()[0]
+            size = int(line.split()[2])
+            if ip_date not in calc_dict:
+                calc_dict[ip_date] = size
+            else:
+                calc_dict[ip_date] = calc_dict[ip_date] + size
+    res_dict = dict()
```

Comment:
I suggest using just "{}" instead of "dict()". This is the convention and there is some argument that dict() is slower
https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/
https://medium.com/@jodylecompte/dict-vs-in-python-whats-the-big-deal-anyway-73e251df8398